### PR TITLE
chat: added DMOptions dropdown to group chat channels

### PR DESCRIPTION
**Pull request by rcrdlbl**
This currently partially solves for #801 

I can send the request to pin a group chat channel, but the backend rejects it (think it'll only take DMs and MultiDMs at this point).

@liam-fitzgerald @arthyn 
mind helping resolve the backend portion of this?

### CODE REVIEW

**Comment by arthyn** [comment]
fixes #801 no actual backend issues, just using the wrong keys

**Comment by jamesacklin** [comment]
This seems fine, @patosullivan review?

File: ui/src/dms/MessagesSidebarItem.tsx [hunk 1]
```diff
@@ -36,14 +40,7 @@ function ChannelSidebarItem({ whom, brief }: MessagesSidebarItemProps) {
     <SidebarItem
       to={`/groups/${groupFlag}/channels/${whom}`}
```
Comment (patosullivan) [inline ui/src/dms/MessagesSidebarItem.tsx:41]:
```suggestion
      to={`/groups/${groupFlag}/channels/${nest}`}
```

If this is set to whom it won't route to the actual channel.

**Review by patosullivan — CHANGES_REQUESTED** [review]
Looks good, just one issue.

**Review by patosullivan — APPROVED** [review]

**Pull request merged**